src/main/java/zd/zero/waifu/motivator/plugin/WaifuMotivatorProject.java

```diff
@@ -119,28 +119,16 @@ private void initializeStartupMotivator() {
             pluginState.isStartupMotivationSoundEnabled()
         );
 
-        doOrElse(
-            VisualMotivationAssetProvider.INSTANCE.createAssetByCategory(
-                WaifuAssetCategory.WELCOMING
+        MotivationFactory.INSTANCE.showMotivationEventForCategory(
```

Comment:
I changed all of the places where the motivation was being constructed and invoked, and replaced it with the unified factory API.